terminfo/dynamic/dynamic.go

```diff
@@ -220,6 +220,9 @@ func LoadTerminfo(name string) (*terminfo.Terminfo, string, error) {
 	t.CursorBack1 = tc.getstr("cub1")
 	t.CursorUp1 = tc.getstr("cuu1")
 	t.SetClipboard = tc.getstr("Ms")
+	if tc.getstr("tsl") != "" && tc.getstr("fsl") != "" {
+		t.SetTitle = tc.getstr("tsl") + "%p1%s" + tc.getstr("fsl")
```

Comment:
These sequences are new to me.  Are they standardized somewhere, because I've never heard of them before.

Reply:
FYI, I'm starting to look at mechanisms to "prebuild" certain escape sequences.  It seems like using $TERM is kind of falling by the way said, and every terminal seems to be mostly implementing the same escapes.

Reply:
> These sequences are new to me. Are they standardized somewhere, because I've never heard of them before.

See `man 5 terminfo` section `Status Lines` or for an older section about it: https://invisible-island.net/ncurses/terminfo.ti.html#tic-xterm_sl (scroll up a bit).

Regarding that section we could actually try to use `TS` first and fall back to `tsl` if that one isn't set.
If we feel fancy we could also check for the availability of `hs` before doing any of that but that feels pretty useless to me, either `TS` / `tsl` and `fsl` are available or they are not.